docs/elements/hx-file-tile/index.html

```diff
@@ -102,6 +102,13 @@ <h2 id="methods">Methods</h2>
         When present, marks element as loading and denotes completion percentage.
       </p>
     </dd>
+
+    <dt>readonly {Boolean} <i>(optional)</i></dt>
+    <dd>
+      <p>
+        When present, user will not be able to dismiss/delete the file.
+      </p>
+    </dd>
   </dl>
 {% endblock %}
 
```

Comment:
Add documentation for the `readonly` property, as well.

Reply:
Additional documentation for the `readonly` property:

        With the `readonly` attribute present, the element will not be dismissible.